Assignments/SCF-Session-2/FCFS/JobScheduler.java

```diff
@@ -0,0 +1,175 @@
+package FCFS;
```

Comment:
Naming of a java package - https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html